content/aips/PRICE-FEEDS-UPDATE-AAVE-V2-V3-04-05-2023.md

```diff
@@ -26,18 +36,31 @@ Upon execution, the proposal will:
 - call `AaveV3Optimism.POOL_ADDRESSES_PROVIDER.setPriceOracleSentinel(0xB1ba0787Ca0A45f086F8CA03c97E7593636E47D5)` to set the price oracle sentinel on the pool addresses provider
 - use Config Engine to update `wstETH` price feed to [0x05225Cd708bCa9253789C1374e4337a019e99D56](https://optimistic.etherscan.io/address/0x05225cd708bca9253789c1374e4337a019e99d56) on Optimism
 - use Config Engine to update `wstETH` price feed to [0x3105C276558Dd4cf7E7be71d73Be8D33bD18F211](https://arbiscan.io/address/0x3105c276558dd4cf7e7be71d73be8d33bd18f211) on Arbitrum
+- use Config Engine to update `MATICX` price feed to [0x0e1120524e14Bd7aD96Ea76A1b1dD699913e2a45](https://arbiscan.io/address/0x3105c276558dd4cf7e7be71d73be8d33bd18f211) on Polygon
+- use Config Engine to update `stMATIC` price feed to [0xEe96b77129cF54581B5a8FECCcC50A6A067034a1](https://arbiscan.io/address/0x3105c276558dd4cf7e7be71d73be8d33bd18f211) on Polygon
 
 ## Security and additional considerations
 
 Custom price adapters are already widely used in the system for price-correlated assets and were [reviewed by the auditors](https://github.com/bgd-labs/cl-synchronicity-price-adapter).
 
+- **MaticX Historical Feeds Comparison**: Comparing the answers from the current feed with the new one for the last 280 days (since CL feed was deployed) with a 4-hour step gave the following results:
```

Comment:
Here you can add that we have also double-checked all points of centralization of MaticX, to verify that no on-chain manipulation of the rate is possible, discounting permissioned actors